docs/nitpick-exceptions

```diff
@@ -1,2 +1,7 @@
 # photutils.morphology
 py:class photutils.morphology.CompoundModel1
+
+# photutils.detection
+py:class photutils.detection.findstars.StarFinderBase
```

Comment:
No, they need to be public to prevent `Sphinx` warnings and to make the class inheritance diagrams.  The aperture base classes are another example in `photutils`:
http://photutils.readthedocs.io/en/latest/api/photutils.aperture.Aperture.html#photutils.aperture.Aperture

See the class inheritance diagram at the bottom of this page:
http://photutils.readthedocs.io/en/latest/photutils/aperture.html